Dockerfiles/midstream/run_tests.py

```diff
@@ -87,13 +100,15 @@ def test_for_extract_and_validate_bundle_image(self):
         return 0
 
 if __name__ == '__main__':
-    resultcodes = [0, 50, 70, 102, 103]
+    import time
+    time.sleep(3600)
```

Comment:
Why do we need to have sleep here?

If you want to debug something you needn't puss that int pull request.

Reply:
removed